kbase-extension/static/kbase/js/common/cellComponents/tabs/results/resultsTab.js

```diff
@@ -81,18 +81,30 @@ define(['bluebird', 'common/events', './outputWidget', './reportWidget'], (
 
             const reports = getReportRefs();
 
+            let spinnerNode = document.createElement('div');
```

Comment:
There is a `loading` method in the `common/ui` module that you can use for this if you want to.

Reply:
Thanks for pointing this out. I updated this to use that, and updated that module to make it a static function (no reason it needs to be tied to a UI object).